extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -799,7 +799,7 @@ export class AmpA4A extends AMP.BaseElement {
           // viewport but cannot wait on promise.  Sadly, need a state a
           // variable.
           this.isVerifiedAmpCreative_ = !!creative;
-          return creative && utf8Decode(creative);
+          return creative && Promise.resolve(utf8Decode(creative));
```

Comment:
This change is unnecessary, the enclosing `then` block will wrap in a promise automatically.